diff --git a/node_modules/astro-agent-annotate/dist/index.js b/node_modules/astro-agent-annotate/dist/index.js
index a4ee475..3e6bda7 100644
--- a/node_modules/astro-agent-annotate/dist/index.js
+++ b/node_modules/astro-agent-annotate/dist/index.js
@@ -36,7 +36,7 @@ function astroAgentAnnotate(options = {}) {
           id: "astro-agent-annotate",
           name: "Agent Annotate",
           icon: ICON,
-          entrypoint: fileURLToPath(new URL("./app.js", import.meta.url))
+          entrypoint: "astro-agent-annotate/app"
         });
       }
     }
@@ -46,4 +46,4 @@ var ICON = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="no
 export {
   astroAgentAnnotate as default
 };
-//# sourceMappingURL=index.js.map
\ No newline at end of file
+//# sourceMappingURL=index.js.map
